pytorch_lightning/trainer/data_loading.py

```diff
@@ -100,6 +102,11 @@ def _worker_check(self, dataloader: DataLoader, name: str) -> None:
                 f' in the `DataLoader` init to improve performance.'
             )
 
+    @staticmethod
+    def auto_add_worker_init_fn(dataloader: DataLoader) -> None:
+        if dataloader.worker_init_fn is None and os.environ.get("PL_SEED_WORKERS", False):
```

Comment:
No need to convert the `environ.get` to int?

Reply:
you should, all `environ` are string

Reply:
shoot! big small mistake.